src/ds/ds6/hw_monitor_extended_buffers.cpp

```diff
@@ -36,80 +27,90 @@ namespace librealsense
         return static_cast<int>(std::ceil(msg_length / 1024.0f));
     }
 
+    hw_monitor_extended_buffers::hwm_buffer_type hw_monitor_extended_buffers::get_buffer_type(command cmd) const
+    {
+        if (cmd.cmd == ds::fw_cmd::GET_HKR_CONFIG_TABLE || cmd.cmd == ds::fw_cmd::SET_HKR_CONFIG_TABLE)
+        {
+            auto calibration_table_size = get_msg_length(cmd);
+            if (calibration_table_size < HW_MONITOR_BUFFER_SIZE)
```

Comment:
Can't see here but I guess `HW_MONITOR_BUFFER_SIZE` = 1024?
If yes than maybe it should be `<=` for standard transaction ?

Reply:
Or you can ask > and inside return get/set
after return standard

Reply:
The truth is the hkr team used te limitation of 1000 bytes instead of 1024 bytes.
But even if this is not 100% correct, this is still better than taking 1024, because we do add some other bytes at the beginning of the message we send there.
I will correct that.

Reply:
If hkr team did not follow the ICD its an issue and we should not align to it.
Lets speak about it tomorow with Evgeni and get a precised definition.
We want it 100% correct   :-)

Reply:
They did follow the ICD 100%